src/content/docs/en/developers/developer-quickstart.mdx

```diff
@@ -26,7 +26,9 @@ Scroll uses ETH as its native currency, which will be needed to pay transaction
 
 To start building on Scroll, we suggest you begin with using our Scroll Sepolia testnet. You'll first need to acquire some testnet ETH. See the [Faucet](/user-guide/faucet) page for tips on getting test tokens on Sepolia. After this, you can bridge your testnet ETH to the Scroll Sepolia Testnet (Layer 2) using our [Sepolia Bridge](https://sepolia.scroll.io/bridge), as described in the [Bridge article](/user-guide/bridge).
 
-For a walkthrough, start with the User Guide's [Setup](/user-guide/setup) page.
+For a walkthrough, start with the User Guide's [Setup](/user-guide/setup) page. We also have a full video tutorial of setting up Metamask, getting testnet tokens, bridging to Scroll, and deploying below.
```

Comment:
Since this is a quickstart, we don't want to introduce information that slows down developers needing to use this as a reference. Videos will be better suited for other areas of the documentation or external guides we link to in an upcoming section..